Cipher_Suites.mediawiki

```diff
@@ -1,6 +1,9 @@
 = Cipher suite correspondence table =
```

Comment:
This table is apparently generated.  I ran the script and it seemed to fail spectacularly, so maybe don't worry about that.  I was hoping that it wouldn't be bitrotten, but I don't think that it's worth fixing right now (there are too many things broken).

Reply:
Oh, good point, I didn't notice the reference to https://github.com/april/tls-table/blob/master/tls-table.py

I'll open an issue about it and we can ignore it for now.

Reply:
Thanks for updating this manually!

It's been stale for the last couple revisions, see #247